lib/components/DraftailEditor.js

```diff
@@ -812,12 +816,14 @@ class DraftailEditor extends Component<Props, State> {
       entityTypes,
       decorators,
       controls,
+      readOnly,
       maxListNesting,
       plugins,
       topToolbar,
       bottomToolbar,
     } = this.props;
-    const { editorState, hasFocus, readOnly } = this.state;
+    const { editorState, hasFocus, readOnlyState } = this.state;
+    const isReadOnly = readOnlyState;
```

Comment:
Here I think this should be `readOnlyState || readOnly`.

Reply:
Fixed.